code/src/rhino.cpp

```diff
@@ -0,0 +1,97 @@
+#include <rhino.hpp>
+
+namespace r2d2::moving_platform {
+
+    rhino_c::rhino_c(
+        r2d2::moving_platform::qik_2s12v10_c &qik_2s12v10_motorcontroller,
+        base_comm_c &comm, hwlib::adc &encode_m0, hwlib::adc &encode_m1)
+        : moving_platform_c(comm),
+          qik_2s12v10_motorcontroller(qik_2s12v10_motorcontroller),
+          motor_encoder_m0(encode_m0),
+          motor_encoder_m1(encode_m1) {
+    }
+    void rhino_c::set_speed(int8_t new_speed) {
```

Comment:
missing newline at the end of a function declaration

Reply:
https://github.com/R2D2-2019/moving_platform/pull/30/commits/828469270908ab8c8bdaf7bc050cf363f1299915